hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -94,11 +73,8 @@ public class HiveTableOperations extends BaseMetastoreTableOperations
    * @return Iceberg property equivalent to the hmsProp. If no such translation exists, the original
    *     hmsProp is returned
    */
-  public static String translateToIcebergProp(String hmsProp) {
-    return ICEBERG_TO_HMS_TRANSLATION.inverse().getOrDefault(hmsProp, hmsProp);
-  }
-
   private final String fullName;
+
```

Comment:
nit: I would guess, that this extra line is not intended

Reply:
Fixed.